### ISSUE TITLE
Ra Hinge gan loss definition

### ISSUE THREAD

**Issue by sujeet-gandhi** [JihyongOh/JSI-GAN#10]
Hi @sooyekim ,
Thanks for the great work
I had a doubt regarding the Ra Hinge Gan loss definition, which has been used [here](https://github.com/JihyongOh/JSI-GAN/blob/master/ops.py#L167)
Shouldn't it be `loss = (real_loss + fake_loss) / 2` , Since it's relativistic average hinge gan loss.
Just for your reference, i saw implementation of RaGan [here](https://github.com/xinntao/BasicSR/blob/master/basicsr/models/esrgan_model.py#L47)
It will be very helpful if you could please clarify.

**Comment by sooyekim** [comment]
Hi @sujeet-gandhi ,

Thanks for the interest in our work!
We had referred to the implementation [here](https://github.com/taki0112/Tensorflow-Cookbook/blob/master/ops.py).
Note that the constant can also be adjusted by loss coefficients (hyperparameters), so as long as there is a hyperparameter tuning stage, the results would be similar.
